### ISSUE TITLE
Scientific Notation is checkable when disbaled

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#110]
@mbarlow12, I am sure this on your radar or todo list, but just in case I am adding this issue about the Scientific Notation checkbox.

When Force Values is not checked, the Scientific Notation checkbox needs to be disabled.

Currently, I can still check the Scientific Notation checkbox when it is visually greyed-out.

According to the ARIA in HTML5 help https://www.w3.org/TR/html-aria/ we should:

> Use the `disabled` attribute on any element that is allowed the disabled attribute in HTML5.
and
> Only use the `aria-disabled` attribute for elements that are not allowed to have a disabled attribute in HTML5 

It is a little unclear to me if HTML5's `disabled` attribute can be applied to a `checkbox`. I see examples with `fieldset`s and `input type="text"`.  We may need to use `aria-disabled="true"` when Forces Values is `unchecked`.

**Comment by mbarlow12** [comment]
@terracoda could you provide some more details on how you were able to check the checkbox? Were you using a screen reader? Was this the built version or from master?

Since this is a common component, @jessegreenberg and I would like to assess whether it's sim-specific or not. Have we not had a sim where a checkbox is disabled yet?

**Comment by terracoda** [comment]
@mbarlow12, great question. Yes, while ussing VoiceOver on Safari I was using the cursor keys to read through the PDOM, and when passing over Force Values, I unchecked it and then I could still move on and check Scientific Notation...just going to verify on Master. I was using a downloaded built version.

**Comment by terracoda** [comment]
Just verified issue on Master. Here's a screen shot after I checked the greyed out checkbox. 
<img width="967" alt="screen shot 2018-10-31 at 10 00 33 pm" src="https://user-images.githubusercontent.com/1902116/47818631-96b36580-dd58-11e8-94ae-229c1af28683.png">

The behaviour of the visual checkbox is correct. It is greyed out when I uncheck Forces Values and I cannot check it.

In the PDOM, however, the checkbox is not disabled, so I can still check and uncheck it.

**Comment by terracoda** [comment]
@mbarlow12, please re-assign if you have more questions :-)

**Comment by terracoda** [comment]
@zepumph, marking onhold. We can fix this bug once we focus is back on this sim.

**Comment by terracoda** [comment]
@zepumph, this issue is no longer on hold.

**Comment by zepumph** [comment]
This should be dealt with in https://github.com/phetsims/sun/issues/519.

**Comment by zepumph** [comment]
@terracoda is this fixed in master now that we made a solution in https://github.com/phetsims/sun/issues/519?

**Comment by terracoda** [comment]
Yes! You fixed it!
<img width="722" alt="Screen Shot 2019-08-01 at 5 41 27 PM" src="https://user-images.githubusercontent.com/1902116/62307381-cc21e680-b483-11e9-9eda-16cbffeaf1a4.png">

In Voice Over it says "dimmed". Not sure what it says in other screen readers.
Closing.
